docs/upgrade.md

```diff
@@ -0,0 +1,117 @@
+# Observability bundle upgrade guide
+
+This document describes the changes that need to be made to the observability bundle to upgrade from one version to another.
+
+## Upgrade from 0.x.y to 1.0.0
+
+### Changes
+
+Release 1.0.0 of the observability bundle brings along a number of breaking changes that need to be addressed when upgrading from 0.x.y.
+
+The main changes are:
+
+- Moving all user configs from under `apps.<appName>.userConfig` from string to regular helm values to `userConfig.<appName>`
+- Renaming of `prometheus-operator-app` to `kube-prometheus-stack`
+- Renaming of `promtail-app` to `promtail`
+
+### Upgrade steps
+
+To ensure a smooth upgrade, you can keep the old and new values along side each other before the upgrade to release 20.0.0 and then remove the old values once the upgrade is complete.
```

Comment:
This doc talks about upgrading observability bundle to release 1.0.0, and here "upgrade to release 20.0.0" seems out of context.
Don't we have a name for the "Releases"? Like "GiantSwarm Platform  release 20.0.0"?

Reply:
Probablye yes

Reply:
DOne thanks